discord/ext/test/backend.py

```diff
@@ -106,7 +106,8 @@ async def start_private_message(self, user_id):
 
         return facts.make_dm_channel_dict(user)
 
-    async def send_message(self, channel_id, content, *, tts=False, embed=None, nonce=None):
+    async def send_message(self, channel_id, content, *, tts=False, embed=None,
+                           nonce=None, allowed_mentions=None):
```

Comment:
allowed_mentions is never used in the function... so I'm not sure what the purpose of adding it as a keyword argument is?

Reply:
@bravosierra99 before adding this line, pytest test fails with this error :

```console
        else:
            data = await state.http.send_message(channel.id, content, tts=tts, embed=embed,
>                                                                     nonce=nonce, allowed_mentions=allowed_mentions)
E           TypeError: send_message() got an unexpected keyword argument 'allowed_mentions'

.venv/lib/python3.7/site-packages/discord/abc.py:891: TypeError
```

Reply:
interesting.. this might be because I have not update to discord 1.4.1 or whatever...  roger thanks